examples/wavefunctions/wavefunctions.py

```diff
@@ -104,6 +102,13 @@ def evolve_trotter_defun(
     euclidean=False,
     callback=None,
     batch_size=1):
```

Comment:
We made this new `tf.vectorized_map` that allows you to add batching to any function and run it in parallel. It should be included in the next tensorflow release. 

No need to change anything, just a heads up.

Reply:
That's cool, although I don't think it applies to this case. Here, "batch" refers to the amount of the computation placed in a single computational graph. The computation itself is still sequential.